symbulate/results.py

```diff
@@ -441,6 +442,104 @@ def standardize(self):
         elif get_dimension(self) > 0:
             return RVResults((np.asarray(self) - mean_) / sd_)
 
+    def median(self):
+        if all(is_scalar(x) for x in self):
+            return np.median(np.array(self))
+        elif get_dimension(self) > 0:
+            return tuple(np.median(np.array(self), 0))
+        else:
+            raise Exception("I don't know how to take the median of these values.")
+
+    def quantile(self, q):
+        if all(is_scalar(x) for x in self):
+            return np.percentile(np.array(self), q*100)
```

Comment:
Please add a space around mathematical operations like +, -, *, /. So this should be `q * 100`, not `q*100`.